Tangem/UIComponents/WalletSelectorItemView/WalletSelectorItemViewModel.swift

```diff
@@ -84,7 +84,12 @@ class WalletSelectorItemViewModel: ObservableObject, Identifiable {
                 case .loading:
                     viewModel.balanceState = .loading
                 case .loaded(let totalBalance):
-                    let formatted = viewModel.balanceFomatter.formatFiatBalance(totalBalance.balance)
+                    guard totalBalance.allTokensBalancesIncluded else {
```

Comment:
я что то полюбил сейчас вот так писать, но это как хочешь)

```suggestion
                    case .loaded(let totalBalance) where totalBalance.allTokensBalancesIncluded
```

Reply:
ну тут проблема, что надо 2 кейса писать, что в одном случае пустая строка, а во втором отформатированный баланс. Не знаю на сколько это удобно будет. Пока так оставлю, наверное